src/db_adapters/receipts.rs

```diff
@@ -17,10 +17,98 @@ pub(crate) async fn store_receipts(
     pool: &Pool<ConnectionManager<PgConnection>>,
     receipts: Vec<&near_indexer::near_primitives::views::ReceiptView>,
     block_hash: &str,
+    no_strict_mode: bool,
 ) {
     let mut skipping_receipt_ids =
         std::collections::HashSet::<near_indexer::near_primitives::hash::CryptoHash>::new();
 
+    let tx_hashes_for_receipts = find_tx_hashes_for_receipts(
+        &pool,
+        receipts.iter().map(|r| r.receipt_id.to_string()).collect(),
+    )
+    .await;
+
+    let mut receipt_models: Vec<models::receipts::Receipt> = receipts
+        .iter()
+        .filter_map(|r| {
+            if let Some(transaction_hash) =
+                tx_hashes_for_receipts.get(r.receipt_id.to_string().as_str())
+            {
+                Some(models::Receipt::from_receipt_view(
+                    r,
+                    block_hash,
+                    transaction_hash,
+                ))
+            } else {
+                warn!(
+                    target: crate::INDEXER_FOR_EXPLORER,
+                    "Skipping Receipt {} as we can't find parent Transaction for it.",
+                    r.receipt_id.to_string()
+                );
```

Comment:
Let's report these as a single warning message in the `else` branch of the `strict_mode` check.